Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -21,117 +17,227 @@ Example:
     a :file:`Documentation/Settings.cfg` you can use an automatic migration
     tool to :ref:`migrate the settings.cfg into a guides.xml <migrate_guides_xml>
 
+..  contents:: Table of Contents:
+    :local:
+
+
 .. _settings-guides:
 
 The `<guides>` tag
 ==================
 
 It is used for general configuration during parsing and rendering.
 
+The following settings can be relevant for TYPO3 themed documentation:
```

Comment:
Please add that this is the root tag., so the structure is clear.

Reply:
done